src/qt/optionsdialog.cpp

```diff
@@ -329,6 +334,23 @@ void OptionsDialog::updateDefaultProxyNets()
     (strProxy == strDefaultProxyGUI.toStdString()) ? ui->proxyReachTor->setChecked(true) : ui->proxyReachTor->setChecked(false);
 }
 
+void OptionsDialog::setActiveAddressType() {
+    switch(g_address_type) {
+    case OUTPUT_TYPE_LEGACY:
+        ui->addressTypeLegacy->setChecked(true);
+        break;
+    case OUTPUT_TYPE_P2SH:
```

Comment:
Rebase issue, .. should this be `OUTPUT_TYPE_P2SH_SEGWIT`?

Reply:
Yes, it won't compile otherwise.

Reply:
@jonasschnelli Fixed, thanks!